HardCodedStringCheckerSharp/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion( "1.4.0.0" )]
-[assembly: AssemblyFileVersion( "1.4.0.0" )]
+[assembly: AssemblyVersion( "1.5.0.0" )]
+[assembly: AssemblyFileVersion( "1.5.0.0" )]
```

Comment:
Good call!  We'll see (I'll merge it such that your continuous will work).  And then we'll know! :smiley: